website/cue/reference/components/sources/base/dnstap.cue

```diff
@@ -81,7 +95,161 @@ base: components: sources: dnstap: configuration: {
 
 			This should not typically needed to be changed.
 			"""
-		required: false
+		relevant_when: "mode = \"unix\""
+		required:      false
+		type: uint: unit: "bytes"
+	}
+	address: {
+		description: """
+			The socket address to listen for connections on, or `systemd{#N}` to use the Nth socket passed by
+			systemd socket activation.
```

Comment:
```suggestion
			`systemd` socket activation.
```

Reply:
Thanks!
I have copied this over from socket docs. Should this be updated there too?